set.mm

```diff
@@ -640139,31 +640229,80 @@ number axioms (add ~ ax-10 , ~ ax-11 , ~ ax-13 , ~ ax-nul , and remove
   $( ~ inelr without ~ ax-mulcom .  (Contributed by SN, 1-Jun-2024.) $)
   sn-inelr $p |- -. _i e. RR $=
     ( ci cr wcel cc0 clt wbr wceq 0re wn c1 cresub co 1re ax-mp wa anidms caddc
-    cmul ax-icn eqtri w3o lttri4 mpan2 reneg1lt0 ltnsymi relt0neg1 mulgt0 sylan
-    rernegcl elre0re resubdi syl3anc remul01 syl remulcld imulid1 eqcomi oveq2i
-    wb reixi oveq12i ax-1cn recni adddii renegid sn-it0e0 eqtr3i renegadd mpdan
-    id mpbiri oveq1d mulassi oveq1i mulcli a1i rei4 3eqtrd oveq12d eqtrd breq2d
-    adantr ex sylbid mtoi 0ne1 neii oveq12 ax-i2m1 remul02 readdid2 3eqtr3g mto
-    mpbid breqtrdi 3ioran syl3anbrc pm2.65i ) ABCZADEFZADGZDAEFZUAZWSDBCZXCHADU
-    BUCWSWTIXAIZXBIXCIWSWTDDJKLZEFZXFDEFXGIUDXFDJBCZXFBCMJUINZHUENZWSWTDDAKLZEF
-    ZXGAUFWSXLXGWSXLODXKXKRLZEFZXGWSXKBCZXLXNAUIZXOXLOXNXKXKUGPUHWSXNXGUSXLWSXM
-    XFDEWSXMXKDRLZXKARLZKLZXFWSXOXDWSXMXSGXPAUJWSVJZXKDAUKULWSXQDXRJKWSXOXQDGXP
-    XKUMUNWSXRAAARLZRLZARLZYAYARLZJWSXKYBARWSYBBCZXKYBGZWSAYAXTWSAAXTXTUOUOWSYE
-    OYFAYBQLZDGYGAJRLZAXFRLZQLZDAYHYBYIQYHAUPUQYAXFARUTURVAAJXFQLZRLZYJDAJXFSVB
-    XFXIVCVDYLADRLDYKDARXHYKDGMJVENURVFTVGTAYBVHVKVIVLYCYDGWSYAARLZARLYCYDYMYBA
-    RAAASSSVMVNYAAAAASSVOSSVMVGVPYDJGWSVQVPVRVSVTWAWBWNWCWDWEXEWSXADJGDJWFWGXAY
-    AJQLDDRLZJQLZDJXAYAYNJQXAYAYNGADADRWHPVLWIYODJQLZJYNDJQXDYNDGHDWJNVNXHYPJGM
-    JWKNTWLWMVPWSXBXGXJWSXBXGWSXBOZDYAXFEYQDYAEFAAUGPUTWOWCWEWTXAXBWPWQWR $.
+    cmul ax-icn a1i w3o lttri4 mpan2 reneg1lt0 rernegcl ltnsymi relt0neg1 sylan
+    mulgt0 elre0re resubdi syl3anc remul01 syl remulcld ipiiie0 renegadd mpbiri
+    wb id mpdan oveq1d mulassi oveq1i mulcli eqtr3i 3eqtrd oveq12d eqtrd breq2d
+    rei4 adantr mpbid ex sylbid mtoi 0ne1 oveq12 ax-i2m1 remul02 readdid2 eqtri
+    neii 3eqtr3g mto reixi breqtrdi 3ioran syl3anbrc pm2.65i ) ABCZADEFZADGZDAE
+    FZUAZWKDBCZWOHADUBUCWKWLIWMIZWNIWOIWKWLDDJKLZEFZWRDEFWSIUDWRDJBCZWRBCMJUENH
+    UFNZWKWLDDAKLZEFZWSAUGWKXCWSWKXCODXBXBRLZEFZWSWKXBBCZXCXEAUEZXFXCOXEXBXBUIP
+    UHWKXEWSUSXCWKXDWRDEWKXDXBDRLZXBARLZKLZWRWKXFWPWKXDXJGXGAUJWKUTZXBDAUKULWKX
+    HDXIJKWKXFXHDGXGXBUMUNWKXIAAARLZRLZARLZXLXLRLZJWKXBXMARWKXMBCZXBXMGZWKAXLXK
+    WKAAXKXKUOUOWKXPOXQAXMQLDGUPAXMUQURVAVBXNXOGWKXLARLZARLXNXOXRXMARAAASSSVCVD
+    XLAAAASSVESSVCVFTXOJGWKVKTVGVHVIVJVLVMVNVOVPWQWKWMDJGDJVQWCWMXLJQLDDRLZJQLZ
+    DJWMXLXSJQWMXLXSGADADRVRPVBVSXTDJQLZJXSDJQWPXSDGHDVTNVDWTYAJGMJWANWBWDWETWK
+    WNWSXAWKWNWSWKWNOZDXLWREYBDXLEFAAUIPWFWGVNVPWLWMWNWHWIWJ $.
   $( $j usage 'sn-inelr' avoids 'ax-mulcom'; $)
 
-  $(
-    Using ~ sn-inelr we can show:
-    ` ( r + _i s ) = ( t + _i u ) <-> ( r = t /\ s = u ) ` but so far there is
-    no use for such a theorem.
-  $)
+  ${
+    $d R x $.
+    $( ` _i ` times a real is non-real iff the real is non-zero.  (Contributed
```

Comment:
Why is the comment negated? "` _i ` times a real is real iff the real is zero." sounds better.